src/routes/give-away/get.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import RestifyErrors from 'restify-errors'
+
+import models from '../../models'
+
+export default {
+
+  method: 'get',
+
+  // @TODO: How to filter?
+  path: '/giveaways?',
+
+  parameters: Joi.object().keys({
+    active: Joi.boolean(),
+  }),
+
+  handler(request, response) {
+
+    // const populateConditions = request.parameters.include.map((active) => {
```

Comment:
This can be removed and you should just auto-populate the metadata, because you'll always need the metadata anyway (see 42948fa).